index.js

```diff
@@ -1,22 +1,18 @@
 var pull = require('pull-stream');
 var toPull = require('stream-to-pull-stream');
 var datDefaults = require('dat-swarm-defaults');
```

Comment:
If this is not going to be used anymore, remove it from here as well as from the package.json

Reply:
Probably a good idea.